frontend/packages/ux-editor/src/hooks/queries/useDatamodelMetadataQuery.ts

```diff
@@ -17,6 +17,7 @@ export const useDatamodelMetadataQuery =
           }
         });
         return dataModelFields;
-      })
+      }),
+      { staleTime: 1000 }
```

Comment:
Jeg ser at `staleTime` settes til `Infinity` i `ServicesContext`. Er ikke helt sikker på om det er hensiktsmessig. Som standard er den jo 0, og jeg ser ikke noen grunn til å gjøre den større, annet enn å spare serveren for arbeid (hvis jeg har forstått `staleTime`-konseptet riktig). Kanskje @mlqn har noen innspill? Ev. kan vi diskutere det når David er tilbake fra ferie, han har jo også erfaring med React Query.
I mellomtiden går jeg god for at vi setter `staleTime` spesifikt her. Alternativt kan vi invalidere eller bare oppdatere mellomlageret når det gjøres endringer på datamodellen. Det er slik vi har løst tilsvarende problemer andre steder, men jeg ser at det fort blir uoversiktlig når vi har så mange datasett som avhenger av hverandre.

Reply:
Enig. Jeg har nesten alltid brukt standardverdiene, så jeg er litt usikker på hva som er de beste verdiene å bruke med tanke på ytelse, men å sette `staleTime` til `Infinity` er et problem fordi det ikke oppdaterer dataene. Det enkleste ville være å bruke standardverdien først ved å fjerne den fra ServicesContext og optimisere den senere hvis det trengs. Å gjøre det vil løse flere andre problemer som vi har med data som ikke er fjernet av cachen (f.eks. #10707).